Doc/library/configparser.rst

```diff
@@ -136,6 +136,26 @@ sections [1]_.  Note also that keys in sections are
 case-insensitive and stored in lowercase [1]_.
 
 
+If we need to read several configurations, each one having more priority than the previous one, we can use the same :class:`ConfigParser` instance to override previous defined data and keep not redefined data.
```

Comment:
I would use "use" instead of "we can use".

Reply:
I reused aeros167's version so do you think this change is needed?
The new version is `...an instance of :class:ConfigParser can be
used to override...`.

Reply:
No problem.